packages/loading-indicator/src/BlobLoader/BlobLoader.types.ts

```diff
@@ -0,0 +1,5 @@
+import { DarkModeProps } from '@leafygreen-ui/lib';
+
+export default interface SpinnerProps extends DarkModeProps {
+  description?: string;
```

Comment:
Is there no inline/horizontal variant for the blob?

Reply:
nope!